test/User-test.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from 'chai';
+import userData from '../src/data/user-test-data';
+import User from '../src/User';
+
+describe('User', () => {
+  let user, userIndex;
+
+  beforeEach(() => {
+    userIndex = 0;
+    user = new User(userData[userIndex]);
+  });
+
+  it('should be a function', () => {
+    expect(User).to.be.a('function');
+  });
+
+  it('should be an instance of User', () => {
+    expect(user).to.be.an.instanceOf(User);
+  });
+
+  it('should be able to store a user ID', () => {
+    expect(user.id).to.equal(1);
+  });
+
+  it('should be able to store a user name', () => {
+    expect(user.name).to.equal('Luisa');
+  });
+
+  it('should be able to store a user address', () => {
+    expect(user.address).to.equal(
+      '15195 Nakia Tunnel, Erdmanport VA 19901-1697'
+    );
```

Comment:
I didn't know this was a solution to break lines into shorter lines! Good catch.